content/en/docs/reference/access-authn-authz/extensible-admission-controllers.md

```diff
@@ -721,7 +721,7 @@ The `matchPolicy` for an admission webhooks defaults to `Equivalent`.
 
 ### Matching requests: `matchConditions`
 
-{{< feature-state state="alpha" for_k8s_version="v1.27" >}}
+{{< feature-state state="beta" for_k8s_version="v1.28" >}}
```

Comment:
Do we want to add information on the limit we set(65) on MatchConditions into the doc?

Reply:
Added a note stating the 64 elements limit